registry/application/services/organization_publisher_service.py

```diff
@@ -53,7 +53,7 @@ def get_org_id_availability_status(self, org_id):
 
         if org_id in org_id_list:
             return OrganizationIDAvailabilityStatus.UNAVAILABLE.value
-        if OrganizationService().is_org_published(org_id, EnvironmentType.MAIN.value):
+        if RegistryBlockChainUtil(EnvironmentType.MAIN.value).is_org_published(org_id):
```

Comment:
Now organization module has dependecy on service_module because of RegistryBlockChainUtil,we need to have blockchian operation for organisation in org module